docs/components/checkbox-group-field/demo/base-demo.md

```diff
@@ -5,20 +5,23 @@
   @name='options'
   @value={{this.groupValue}}
   @onChange={{this.updateValue}}
-  as |group|
 >
-  <group.CheckboxField @label='Option 1' @value='option-1' />
-  <group.CheckboxField
-    @label='Option 2'
-    @hint='Extra information about the radio'
-    @value='option-2'
-  />
-  <group.CheckboxField @label='Option 3' @value='option-3' />
-  <group.CheckboxField
-    @label='Option 4'
-    @hint='Extra information about the radio'
-    @value='option-4'
-  />
+  <:default as |group|>
+    <group.CheckboxField @label='Option 1' @value='option-1' />
+    <group.CheckboxField
+      @label='Option 2'
+      @hint='Extra information about the radio'
+      @value='option-2'
+    />
+    <group.CheckboxField @label='Option 3' @value='option-3' />
+    <group.CheckboxField
+      @label='Option 4'
+      @hint='Extra information about the radio'
+      @value='option-4'
+    />
+  </:default>
+  <:label>extra label info</:label>
+  <:hint>extra hint info</:hint>
```

Comment:
I'm wondering:

Should the named-block options go under a new section + under the UI states sections instead?  I feel like they could use some extra documentation and that'd be a good way to show usage + what they look like visually.

Reply:
These `:label` and `:hint` blocks are removed from this demo now (they should not have been here).

I also added `:label` code snippets under each `## Label` section, same for `:hint`. I don't think they can go under UI states because folks can't see the code there...I guess they could see an icon for label or a link for hint? Seems more useful to have it under the `## Label` and `## Hint` sections tho (in terms of being able to copy those code snippets for consumers)